taskotron_python_versions/common.py

```diff
@@ -1,7 +1,55 @@
 import logging
+import os
+
+import rpm
 
 log = logging.getLogger('python-versions')
 log.setLevel(logging.DEBUG)
 log.addHandler(logging.NullHandler())
 
 BUG_URL = 'https://github.com/fedora-python/task-python-versions/issues'
+
+
+class PackageException(Exception):
+
+    """Base Exception class for Package API."""
+
+
+class Package(object):
+
+    """RPM Package API."""
+
+    def __init__(self, path):
+        """Given the path to the RPM package, initialize
+        the RPM package header containing its metadata.
+        """
+        self.filename = os.path.basename(path)
+        # To be populated in the first check.
+        self.py_versions = []
```

Comment:
This initial value is weird. A dict is assigned in `check_two_three` for py_versions so we might use an empty dict here instead. But that would be confusing with and actual result, that can also be an empty dict. Let's go with `None` to indicate "not yet known".